ethereum-consortium-blockchain-network/nested/fiveBootNodeVMExtension.json

```diff
@@ -73,13 +73,15 @@
       "properties": {
         "publisher": "Microsoft.OSTCExtensions",
         "type": "CustomScriptForLinux",
-        "typeHandlerVersion": "1.4",
+        "typeHandlerVersion": "2.0",
```

Comment:
@singhkays  @cavanes this is not going to work. You should be using `Microsoft.Azure.Extensions` `CustomScript`.